### DISCUSSION TITLE
TypeError: Failed to fetch

### DISCUSSION THREAD

**Discussion by lianzhengyun** [nextai-translator/nextai-translator#614] [Q&A]
使用自定义API，出现TypeError: Failed to fetch

**Comment by tallietian** [comment]
我也遇到了相同的问题

**Comment by maltoze** [comment]
![image](https://github.com/openai-translator/openai-translator/assets/18044730/0a3656d0-810f-4f2b-8952-4d4bb214686e)

**Reply by tallietian** [reply]
啥意思呢，没看懂。。

**Reply by maltoze** [reply]
我用自定义api测了下，没有报错

**Reply by tallietian** [reply]
是桌面端报错了，不是chrome扩展

**Comment by maltoze** [comment]
可以试试在服务器上加上这个返回头

![image](https://github.com/openai-translator/openai-translator/assets/18044730/55ca8d3e-5d7e-4b88-a626-c9ee80427706)

**Reply by easeaico** [reply]
把请求地址改成http://127.0.0.1:3333试试

**Comment by BloodGrape** [comment]
求解决方法
